src/events/account_events.rs

```diff
@@ -0,0 +1,141 @@
+//! contains tests for account (list) events
```

Comment:
Better to put this into `src/tests/account_events.rs` as done for verified chats to struct everything uniformely